tests/microos/toolbox.pm

```diff
@@ -39,6 +39,16 @@ sub run {
     select_console 'root-console';
     $self->create_user;
 
+    my $toolbox_image_to_test = get_var('CONTAINER_IMAGE_TO_TEST');
+
+    if ($toolbox_image_to_test) {
+        (my $registry = $toolbox_image_to_test) =~ s/\/.*//;
```

Comment:
A small comment describing what those cryptic regexes are would be very helpful for our future selves as regexes are typically write-only :wink:

Reply:
done